test/fuzz/basic.ml

```diff
@@ -87,10 +87,10 @@ let ibinop =
     ; const (Xor : ibinop)
     ; const (Shl : ibinop)
     ; shr
-    ; const (Rotl : ibinop)
-    ; const (Rotr : ibinop)
     ]
 
+let ibinop_rot = choose [ const (Rotl : ibinop); const (Rotr : ibinop) ]
```

Comment:
Why are we separating these now ?

Reply:
OK, see my comment on the PR below then.

Reply:
This is my fault, it should be easy to implement these as it's only the concrete evaluation case that is missing https://github.com/formalsec/encoding/issues/79

Reply:
Oh they exist in encoding ? I though it was like the `copysign` one, which was easy to implement symbolically directly in owi (it is exactely the same code as the concrete version but with symbolic primitives)

Reply:
This is the reverse case, they are mapped to `z3` but not resolved concretely in `unop` in `symbolic_value.ml` which calls encoding's `eval_numeric.ml` but I will push a fix in a bit for `Rot(l|r)` and `Trunc`.

Reply:
For the concrete case, you could just copy what's in `owi/src/int32.ml` right ?

Reply:
> For the concrete case, you could just copy what's in owi/src/int32.ml right ?
 
Yes I will just plagiarize it :sweat_smile: